doc/sphinx-guides/source/developers/big-data-support.rst

```diff
@@ -100,6 +122,7 @@ Optional steps for setting up the S3 Docker DCM Variant
 ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
 
 - Before: the default bucket for DCM to hold files in S3 is named test-dcm. It is coded into `post_upload_s3.bash` (line 30). Change to a different bucket if needed.
+- Also Note: With the new support for multiple file store in Dataverse, DCM requires a store with id="s3" and DCM will only work with this store.
```

Comment:
Heads up to @pameyer about this if we merge this. Affects DCM.

Reply:
FWIW: As long as there's a store with the id 's3', as recommended for backward compatibility, DCM should be unaffected/ will continue to work on an instance that upgrades from a single s3 store today.

In looking through the DCM code, I did see that it creates an AWS client without using any of the s3 store options that have (since?) been created to support minio, etc. Given that, I think DCM can only currently work with a vanilla AWS S3 server and this branch won't affect that.